src/common/home/Home.jsx

```diff
@@ -31,6 +36,34 @@ const Home = () => {
     });
   }, [data, setSearchTerm, searchTerm, setFilterQuery]);
 
+  //* array of tweet IDs to show on the home page
+    const tweetIdArray = [
+        "1544376341172068352",  
+        "1530197614771458049",
+        "1529884210269671424",
+        "1522165831005728769",
+        "1521784126717710336",
+        "1518200201302974464",
+        "1541802575775035392",
+        "1515598358748237830",
+        "1528938773710782464",
+        "1532349503709122561"
+    ]
+
+    //* get a single wrapper for one of the embeded tweets
+    const ele = document.querySelector(".tweets-container")?.childNodes[0]?.childNodes[0]
+    const [tweetWrapperElement,setTweetWrapperElement] = useState(undefined)
+
+    useEffect(()=>{
+        setTimeout(()=>{
```

Comment:
@Anirban-1490 I don't think we need it. The setTimeout doesn't have any guerntee that the DOM will be loaded within 8 sec(which is huge btw) time you have given.

I would suggest you remove it. Only way I can think of is using the useRef and check if the component is mounted. I don't see a need of that as it was loading very fast. Kindly make the changes.

Reply:
> @Anirban-1490 I don't think we need it. The setTimeout doesn't have any guerntee that the DOM will be loaded within 8 sec(which is huge btw) time you have given.
> 
> I would suggest you remove it. Only way I can think of is using the useRef and check if the component is mounted. I don't see a need of that as it was loading very fast. Kindly make the changes.

well even using useRef i can only put it in the parent container which get's mounted instantly ,so i mean there is still no point of using it i think. If i can only get to the bottom i mean in the TWEET Iframe then only this will be valid but as i said before i tried many times even with useRef but it always gives me infinite re-renders. But i will try once again today

Reply:
Let's remove the setTimeout as the 8sec delay is unrealistic.